src/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Meal Mentor</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+      integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="logo-div">
+        <a href="index.html"
+          ><img
+            class="img-logo"
+            src="./assets/logo-transparent.png"
+            alt="meals"
+        /></a>
+        <h1 class="header-title">Meal Mentor</h1>
+      </div>
+      <h2 class="title-recipe">Recipes</h2>
+      <nav class="nav">
+        <ul class="list">
+          <li><a class="serve" href="#">Recipes @ your pocket</a></li>
+          <a href=""
+            ><img
+              class="bell"
+              src="./assets/reception-bell-svgrepo-com.svg"
+              alt="#"
+          /></a>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section>
+        <div id="meals" class="meals"></div>
+      </section>
+
+      <section class="popup-container hidden">
+        <div class="popup-window">
+          <div class="close">
```

Comment:
- [ ] [OPTIONAL] For a better user interface(UI), I would recommend that you move the close button to the extreme right by adding some margin/padding.
![Screenshot from 2022-09-17 20-01-04](https://user-images.githubusercontent.com/58692590/190872648-1fd82a76-00ae-4cb3-9632-7244d726120c.png)

Reply:
Done